.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "endOfLine": "auto",
+    "printWidth": 100,
+    "bracketSpacing": true,
+    "semi": false,
```

Comment:
As long as you changed the rule to disable semicolons, all files containing semicolons should be changed by this new prettier rule.

I don't see all the other file changes by removing all the semicolons.

Reply:
Keep all the semicolons & delete "semi": false